front/public/locales/de/operationalStudies/drivertrainschedule.json

```
@@ -0,0 +1,27 @@
+{
+  "actualspeed": "Tatsächlich",
+  "actualspeed-short": "Tat.",
+  "averagespeed": "Durchschnitt",
+  "averagespeed-short": "Dur.",
+  "composition": "Zusammensetzung",
+  "dcmdetails": "Zugverlauf n°",
+  "destination": "Ziel",
+  "energyconsumed-basic": "Kürzeste Laufzeit",
+  "energyconsumed-eco": "Energieschonendste Laufzeit",
+  "energydetails": "Energieumsatz am Radumfang",
+  "mass": "Masse",
+  "numberoflines": "Linienanzahl",
+  "origin": "Ursprung",
+  "pk": "PK",
```

```suggestion
  "pk": "BP",
```

In german, I would say "Betriebspunkt", this is what uses [Netzgrafik-Editor](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend)